templates/crds/addressplans.crd.yaml

```diff
@@ -72,13 +78,27 @@ spec:
                   type: number
                 broker:
                   type: number
-            messageTtl:
-              type: object
-              description: "Restricts message time-to-live (TTL). Applies to address types queue and topic only."
-              properties:
-                maximum:
-                  description: "Imposes a maximum TTL value (in milliseconds) for messages arriving at the address.  Messages arriving with TTL values greater than this value (or no TTL value) will have their expiration adjusted."
-                  type: integer
-                minimum:
-                  description: "Imposes a minimum TTL value (in milliseconds) for messages arriving at the address.  Messages arriving with TTL values less than this value will have their expiration adjusted."
-                  type: integer
+        displayName:
```

Comment:
I think the intent was to re-add the old top level fields, did you intent to remove the messageTtl fields?

Reply:
Thanks @k-wall , that was an oversight!